shared/chat/conversation/info-panel/index.desktop.js

```diff
@@ -1,19 +1,19 @@
 // @flow
 import * as React from 'react'
-import {Box, Button, Checkbox, Divider, Icon} from '../../../common-adapters'
+import {Avatar, Box, Button, Checkbox, Divider, Icon, Text} from '../../../common-adapters'
 import {globalColors, globalMargins, globalStyles} from '../../../styles'
 import Participants from './participants'
 
-import type {Props} from '.'
+import type {InfoPanelProps, ChannelInfoPanelProps} from '.'
 
 const border = `1px solid ${globalColors.black_05}`
-const InfoPanel = (props: Props) => (
+const InfoPanel = (props: InfoPanelProps) => (
```

Comment:
I looked at your PR, and I'm not sure SmallTeamInfoPanel and BigTeamInfoPanel are the right names for these. You can have a pretty big ad-hoc channel and a small team channel, right?